dev/run-tests-jenkins.py

```diff
@@ -39,7 +39,8 @@ def print_err(msg):
 def post_message_to_github(msg, ghprb_pull_id):
     print("Attempting to post to Github...")
 
-    url = "https://api.github.com/repos/apache/spark/issues/" + ghprb_pull_id + "/comments"
+    api_url = os.getenv("GITHUB_SERVER_API_URL", "https://api.github.com/repos/apache/spark")
```

Comment:
How about `GITHUB_API_BASE`? This is because [another script](https://github.com/apache/spark/blob/master/dev/github_jira_sync.py#L35) uses this name for this URL.

Reply:
Sure. @kiszk